src/go/options/configgenerator.go

```diff
@@ -35,6 +35,7 @@ type ConfigGeneratorOptions struct {
 	CorsAllowOrigin      string
 	CorsAllowOriginRegex string
 	CorsExposeHeaders    string
+	CorsMaxAge           string
```

Comment:
Either keep as `int` or change to `time.Duration`. No need for it to be a string. You can convert to `string` in `router_generator.go`.

Reply:
Done.